torch/_refs/__init__.py

```diff
@@ -2038,10 +2039,27 @@ def tensor_split(
         return tuple(splits)
 
 
+def t(a: TensorLikeType):
+    # Currently refs don't support sparse checks
```

Comment:
I was a little confused about why this comment is here, maybe something like

```
# TODO: enable sparse support 
```

would be clearer?

Reply:
Sounds good.